decidim-core/lib/decidim/db/common-passwords.txt

```diff
@@ -5543,7 +5543,7 @@ capricorn1
 capilla951
 canoncanon
 candycandy
-can'tremember
+cannotremember
```

Comment:
Are we sure we want to change this file ?

Reply:
No, we don't want to change that. Fixed by 8b10d82.